stories/forms/form.stories.js

```diff
@@ -17,7 +17,8 @@ import {
   Toggle,
   Radio,
   RadioGroup,
-  Range
+  Range,
+  Checkbox
```

Comment:
I'd like to keep this list in alphabetical order to make it easier to scan and find things, and less subjective as to where to place new items - can you move this up to the right slot based on that please?

Reply:
Sorted